components/widgets/menu/menu_items/submenu_item.test.tsx

```diff
@@ -23,13 +21,7 @@ describe('components/widgets/menu/menu_items/submenu_item', () => {
             />,
         );
 
-        expect(wrapper.html()).toEqual(format(`<li class="SubMenuItem MenuItem" role="menuitem" id="1_menuitem">
-                <div class="" id="1" tabindex="0"><span class="MenuItem__primary-text">test
-                    <span id="channelHeaderDropdownIconRight_1" class="fa fa-angle-right SubMenu__icon-right-empty" aria-label="submenu icon"></span></span>
-                    <ul class="a11y__popup Menu dropdown-menu SubMenu" style="visibility: hidden; right: 100%;"></ul>
-                </div>
-            </li>`),
-        );
+        expect(wrapper.html()).toMatchSnapshot();
```

Comment:
If we're removing the inline snapshots here, should we also change the `wrapper.html()` back to just `wrapper` then?